lib/customer-deposit-wallet/src/Cardano/Wallet/Deposit/Pure.hs

```diff
@@ -135,6 +136,9 @@ isCustomerAddress :: Address -> WalletState -> Bool
 isCustomerAddress address =
     flip Address.isCustomerAddress (Read.toRawAddress address) . addresses
 
+fromRawCustomer :: Word31 -> Customer
+fromRawCustomer = fromIntegral
```

Comment:
```suggestion
fromRawCustomer = id
```

As much as I love abstract data types — this one is positively a type synonym. 🤓

https://github.com/cardano-foundation/cardano-wallet-agda/blob/5a24dca9b1d884688483ad78a00b8f7485d522a4/lib/customer-deposit-wallet-pure/agda/Cardano/Wallet/Deposit/Pure/Address.agda#L82

Reply:
indeed! done